src/utils/git.cpp

```diff
@@ -50,12 +56,59 @@ void Utils::Git::commitCurrentNoteFolder() {
     if (!executeGitCommand(git, QStringList{"init"}, process) ||
         !executeGitCommand(git, QStringList{"config", "commit.gpgsign", "false"}, process) ||
         !executeGitCommand(git, QStringList{"add", "-A"}, process) ||
-        !executeGitCommand(git, QStringList{"commit", "-m", "QOwnNotes commit"}, process)) {
+        !executeGitCommand(git, QStringList{"commit", "-m", "QOwnNotes commit"}, process, false, true)) {
+        if ( mw != nullptr ) {
```

Comment:
Can you please use the same coding style for brackets and spaces like in the rest of the project. 😉